### DISCUSSION TITLE
Plugin Unavailable

### DISCUSSION THREAD

**Discussion by msharoff** [grafana/grafana-csv-datasource#226] [Q&A]
Hi,

I am getting a Plugin unavailable after installing the .csv plugin.  It is listed as "signed" when I look at my installed plugins.  I was able to successfully set the data source to my local .csv file with no errors.

Any ideas to try?

Thank you,

Mitch

**Comment by dp106** [comment]
Same issue for me, no luck solving so far :(

**Comment by dp106** [comment]
Ok, so in my case I looked in the Grafana log and it was complaining it could not find an exe file when trying to load. I checked the plugin location and there was indeed no exe file. Came back to github to try and work out why I didn't have the file when I worked out I had loaded the wrong version of the plugin.  Loaded the correct plugin and I've got a bit further, now it won't recognise my file location :)
